redis/connection.py

```diff
@@ -1371,7 +1381,13 @@ def _checkpid(self):
             finally:
                 self._fork_lock.release()
 
-    def get_connection(self, command_name, *keys, **options):
+    def get_connection(
+        self,
+        command_name: str,
+        # TODO: Type the next two parameters.
+        *keys,
+        **options
+    ) -> Connection:
```

Comment:
I couldn't figure out what types the x and y parameters take. If anyone knows please tell me.